UPGRADING.md

```diff
@@ -308,6 +308,19 @@ The following module has moved within the file system: tmgmt_poetry.
   assign it to user "Support SMT Jira", send the ticket number to
   "COMM EUROPA MANAGEMENT"
 
+ * From ticket NEPT-391
+
+```
+$ drush en nexteuropa_varnish  -y
```

Comment:
I think that we should mention here that there are variables which are required to perform the drush command successfully, especially because of the '-y' parameter.